otherlibs/stdune/src/ansi_color.ml

```diff
@@ -30,9 +34,11 @@ module RGB24 : sig
   val red : t -> int
   val green : t -> int
   val blue : t -> int
-  val create : r:int -> g:int -> b:int -> t
+  val make : red:int -> green:int -> blue:int -> t
+  val to_int : t -> int
+  val of_int : int -> t
```

Comment:
For consistency with other functions in the code base, this was renamed to `make` and full labelled arguments given.